src/runtime/manifest-parser.pegjs

```diff
@@ -1668,15 +1668,12 @@ ExpressionWithQualifier
   }
 
 PaxelExpression
-  = expr:(NewExpression / ExpressionWithQualifier) {
+  = expr:(NewExpression / ExpressionWithQualifier / RefinementExpression) {
```

Comment:
keep in mind that this now allows the parser to accept

foo: writes Bar { ... } = 1 + 2

The particle connection handle used to enforce that the RHS has to be either a New Expression or ExpressionWithQualifier.

Reply:
I am comfortable with this - IMO "1 + 2" is a valid paxel expression and should be accepted here in terms of syntax. Once we develop type checking for Paxel we will be catching this as.